pygmt/base_plotting.py

```diff
@@ -440,6 +440,187 @@ def grdview(self, grid, **kwargs):
+
+                Anisotropy bars. barscale sets the scaling of the bars.
+                Parameters are expected to be in the following columns:
+                    **1,2** : longitude, latitude of station (-: option interchanges
+                    order) **3,4** : eastward, northward components of anisotropy
+                    vector (-: option interchanges order)
+
+            "**-Sr**\ *velscale/confidence*\ [\ **+f**\ font]"
+
+                Velocity ellipses in rotated convention. velscale sets the
+                scaling of the velocity arrows. The confidence sets the
+                2-dimensional confidence limit for the ellipse, e.g., 0.95 for
+                95% confidence ellipse. font sets the font and size of the text
+                [9p,Helvetica,black]. The ellipse will be filled with the color
+                or shade specified by the -G option [default transparent].
+                The arrow and the circumference of the ellipse will be drawn
+                with the pen attributes specified by the -W option. Parameters
+                are expected to be in the following columns:
+                    **1,2**: longitude, latitude, of station (-: option interchanges
+                    order) **3,4**: eastward, northward velocity (-: option
+                    interchanges order) **5,6**: semi-major, semi-minor axes
+                    **7**: counter-clockwise angle, in degrees, from horizontal
+                    axis to major axis of ellipse. 8: name of station (optional)
+
+            "**-Sw**\ *wedgescale/wedgemag*"
+
+                Rotational wedges. wedgescale sets the size of the wedges.
+                Values are multiplied by wedgemag before plotting. For example,
+                setting wedgemag to 1.e7 works well for rotations of the order
+                of 100 nanoradians/yr. Use **-G** to set the fill color or shade
+                for the wedge, and **-E** to set the color or shade for the
+                uncertainty. Parameters are expected to be in the
+                following columns:
+                    **1,2**: longitude, latitude, of station (-: option
+                    interchanges order) **3**: rotation in radians **4**: rotation
+                    uncertainty in radians
+
+            "**-Sx**\ *cross_scale*"
+
+                gives Strain crosses. cross_scale sets the size of the cross.
+                Parameters are expected to be in the following columns:
+                    **1,2**: longitude, latitude, of station
+                    (-: option interchanges order) 3: eps1, the most extensional
+                    eigenvalue of strain tensor, with extension taken positive.
+                    **4**: eps2, the most compressional eigenvalue of strain
+                    tensor, with extension taken positive.
+                    **5**: azimuth of eps2 in degrees CW from North.
+
+
+        Other Parameters
+        ----------------
+        -A : bool or str
```

Comment:
```suggestion
        A : bool or str
```

A hyphen isn't needed for the parameters, same with the ones below. Also we're trying to use more descriptive long-form arguments (see #473) in the Python world (see some of the other docstrings in the same file, except `plot` :wink:).